### PR TITLE
Clarify README and make lint command more convenient

### PR DESCRIPTION
**Pull request by gaoxk**
- clarified effects of docker-compose down --volume in readme
- made backend lint command more copy-past friendly in pr template

### CODE REVIEW

**Comment by jennchenn** [comment]
PR template change looks good! Are there any README changes?

**Comment by gaoxk** [comment]
je suis un fool 🤦 its added now

File: README.md [hunk 1]
```diff
@@ -1,46 +1,96 @@
...
 
-Seed your database with the provided script. Before running, you will need to set environment variables for all of the user secrets, which can be found in the secrets channel.
+Insert test data into the database with the provided script. Your environment variables will need to be correctly configured in `.env`
+
 ```
-./seed_db.sh
+docker exec -it planet-read_py-backend_1 /bin/bash -c "python -m tools.insert_test_data"
 ```
 
 If there are no tables in the DB, go into `/backend/python/app/models/__init__.py` and change the `erase_db_and_sync = False` to True, allow the hot reload to build, and change it back to `False`. Try to seed the database again.
```
Comment (jennchenn) [inline README.md:56]: they can also run `flask db upgrade` to create the tables--should it be included in here as well (it is below in the Database Migrations section)?

**Review by jennchenn — APPROVED** [review]
looks good to me! just a quick question about whether we want to include the migration command earlier in the README but looks good to merge either way

File: README.md (hunk 1 above)
Comment (gaoxk) [reply README.md:56]: yep in the db migration section too!

**Review by jennchenn — APPROVED** [review]

**Review by lynn-kim — APPROVED** [review]

**Pull request merged**